Templates/Kotlin/Operation.stencil

```diff
@@ -13,7 +13,7 @@ import javax.annotation.Generated
 
 @Generated("{% if moduleName %}{{ moduleName }}{% else %}com.shopify.syrup{% endif %}")
 {% endif %}
-class {{ name }}({{ operation.variables|renderKotlinArguments }}): {% if isQuery %}Query{% else %}Mutation{% endif %}<{{ operation.name }}Response> {
+class {{ operation|renderClassName }}({{ operation.variables|renderKotlinArguments }}): {{ operation|renderOperationTypeName }}<{{ operation.name }}Response> {
```

Comment:
We have `capitalizeFirstLetter` we could use to format `typeName` correctly from template.

Reply:
I removed `typeName`

Reply:
Right - oops. 👍

Down the road I think we'll want to decide how opinionated we want to be with regards to what we do in templates and what we do in code. Tough to balance but we want a consumer to be able to configure their templates as much possible without touching Syrup source. ... or at least that was the original intention.

Reply:
might be tricky when the templates mysteriously cant render enums

Reply:
Spoke with @g-Off once about dreams of using Liquid at one point. Any thoughts on where the line should be drawn Geoff?